.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -1,30 +1,35 @@
 name: Publish PyTorch Dev Dockers
+
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 2 * * *" # Runs nightly at 2 AM UTC
+    - cron: "0 2 * * *" # Runs Nightly at 2 AM UTC
+
 jobs:
   build-and-push-image:
     strategy:
       fail-fast: false
       matrix:
         targets:
           - amdgpu_target: "gfx942"
-          - amdgpu_target: "gfx1100"
-          - amdgpu_target: "gfx1201"
```

Comment:
I think building for those targets shouldn't be dropped, even though you wont run verification for them.